### PR TITLE
Add Pictograms

### PR DESCRIPTION
**Pull request by jchesterman**
This PR adds a new WWW Site category within Storybook, and adds new Pictograms.

<details>
  <summary>📦 Published PR as canary version: <code>7.15.1-canary.239.5367.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @apollo/space-kit@7.15.1-canary.239.5367.0
  # or 
  yarn add @apollo/space-kit@7.15.1-canary.239.5367.0
  ```
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,4 +143,4 @@
     "react": ">16",
     "framer-motion": ">1.6"
   }
-}
+}
```
Comment (justinanastos) [inline package.json:146]: NBD: I'm assuming this was an inadvertent change; apparently I never set up an `.editorconfig` for Space Kit 🤷‍♂️

File: src/pictograms/pictograms.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import fs from "fs";
+// import { promisify } from "util";
```
Comment (justinanastos) [inline src/pictograms/pictograms.spec.tsx:4]: 🧹 
```suggestion
```

**Review by justinanastos — APPROVED** [review]
I left a few minor comments; which are requests for changes that I'd like done but don't need to verify after you've done them and before you merge.

I should copy this here or find a way to share it: https://github.com/mdg-private/studio-ui#reviewer-guidelines

**Comment by apollo-bot2** [comment]
:rocket: PR was released in [`v7.16.0`](https://github.com/apollographql/space-kit/releases/tag/v7.16.0) :rocket:

**Pull request merged**